src/intl/en/page-learn.json

```diff
@@ -1,37 +1,37 @@
 {
   "toc-learn-hub": "Learn hub",
-  "toc-what-is-crypto-ethereum": "What is crypto and Ethereum?",
+  "toc-what-is-crypto-ethereum": "What is Ethereum?",
   "toc-how-do-i-use-ethereum": "How do I use Ethereum?",
   "toc-what-is-ethereum-used-for": "What is Ethereum used for?",
   "toc-strengthen-the-ethereum-network": "Strengthen the Ethereum network",
-  "toc-learn-about-the-ethereum-protocol": "Learn about Ethereum the protocol",
-  "toc-learn-about-the-ethereum-community": "Learn about Ethereum the community",
+  "toc-learn-about-the-ethereum-protocol": "Learn about the Ethereum protocol",
+  "toc-learn-about-the-ethereum-community": "Learn about the Ethereum community",
   "toc-books-and-podcasts": "Books and podcasts",
   "hero-header": "Learn about Ethereum",
   "hero-subtitle": "Your educational guide to the world of Ethereum. Learn how Ethereum works and how to connect to it. This page includes technical and non-technical articles, guides, and resources.",
   "hero-button-lets-get-started": "Let's get started",
-  "what-is-crypto-1": "You may have heard about cryptocurrencies, Bitcoin, and blockchain. This page will help you learn about what they are how they relate to Ethereum.",
-  "what-is-crypto-link-1": "And what is Ethereum anyway?",
-  "what-is-crypto-2": "Bitcoin enables anyone to transfer money globally. Ethereum does too, <b>and</b> it enables anyone to create and agree on any kind of contract they can imagine. It’s both resilient and flexible: any computer program can run on Ethereum.",
+  "what-is-crypto-1": "You may have heard about cryptocurrencies, blockchains and Bitcoin. The links below will help you to learn what they are and how they relate to Ethereum.",
+  "what-is-crypto-2": "Cryptocurrencies, such as Bitcoin, enable anyone to transfer money globally. Ethereum does too, but it can also run code that enables people to create apps and organizations. It’s both resilient and flexible: any computer program can run on Ethereum. Learn more and find out how to get started:",
   "what-is-ethereum-card-title": "What is Ethereum?",
   "what-is-ethereum-card-description": "If you are new, start here to learn why Ethereum matters.",
   "what-is-ethereum-card-image-alt": "Illustration of a person peering into a bazaar, meant to represent Ethereum.",
   "what-is-eth-card-title": "What is ETH?",
   "what-is-eth-description": "Ether (ETH) is the currency powering the Ethereum network and apps.",
-  "where-can-i-get-eth-card-title": "Where can I get ETH?",
```

Comment:
Don't understand this change. This is one of the top questions new users have and we shouldn't bury it in the 'other links' section IMO.

Reply:
The cards are supposed to link to content that answer the question in their subheading: "what is Ethereum?" 
The three new cards all contribute to the answer. The 'Get ETH' page doesn't - it tells people how to get it but doesn't help them understand what it is.